src/components-examples/cdk/stepper/cdk-linear-stepper-with-form/example-custom-linear-stepper.html

```diff
@@ -0,0 +1,22 @@
+<section class="example-container">
+    <header>
+        <h2>Step {{ selectedIndex + 1}}/{{ steps.length}}</h2>
+    </header>
+
+    <div [style.display]="selected ? 'block' : 'none'">
+        <ng-container [ngTemplateOutlet]="selected.content"></ng-container>
```

Comment:
You should be able to put the `ngTemplateOutlet` on the `div` directly which will allow you to drop the `ng-container`.

Reply:
Yep this worked.